properties.schema

```diff
@@ -419,84 +429,84 @@
             "type": "string",
             "inputType": "ColourPicker",
             "default": "#36CDE8",
-            "help": "Notify background colour"
+            "help": "Sets the Notify (pop up) colour."
           },
           "notify-inverted": {
             "title": "Notify colour - inverted",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "#FFFFFF",
-            "help": "Notify foreground colour"
+            "help": "Defines the Notify text colour. Should be a colour that provides good contrast against the Notify colour."
           },
           "notify-link": {
             "title": "Notify link colour",
             "type": "string",
             "inputType": "ColourPicker",
             "default": "",
-            "help": "Notify link font colour"
+            "help": "Defines the Notify link font colour."
```

Comment:
I would change the title to 'Notify link font colour' and then drop the help text